vhdbuilder/packer/components.json

```diff
@@ -199,17 +189,13 @@
       "downloadURL": "mcr.microsoft.com/oss/cilium/operator-generic:*",
       "amd64OnlyVersions": [],
       "multiArchVersions": [
-        "1.13.10",
         "1.14.4"
       ]
     },
     {
       "downloadURL": "mcr.microsoft.com/oss/cilium/cilium:*",
       "amd64OnlyVersions": [],
       "multiArchVersions": [
-        "1.13.10-1",
-        "1.13.10-2",
-        "1.14.4",
```

Comment:
Check with Camryn and Tamilmani. I think we can safely remove 1.13.10-1 and 1.14.4-1, but 1.13.10-2 is still used by K8s 1.27. It wouldn't break anything to remove it, but it could affect node startup time.

Reply:
Have pinged.